helper/pluginutil/tls.go

```diff
@@ -146,19 +146,19 @@ func VaultPluginTLSProvider(apiTLSConfig *api.TLSConfig) func() (*tls.Config, er
 
 		addrRaw := wt.Claims().Get("addr")
 		if addrRaw == nil {
-			return nil, errors.New("decoded token does not contain primary cluster address")
+			return nil, errors.New("decoded token does not contain primary cluster api_address")
```

Comment:
This error message isn't really good (originally, not due to your change) -- it's not really "primary cluster" it's "active node". But really just, the originating Vault process.